src/components/app/App.js

```diff
@@ -14,9 +18,16 @@ function App() {
     <div className="App">
       <Header />
       <div>content will appear here soon</div>
-      {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
+
+        {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
+        <div>
```

Comment:
You don't need this wrapping div here and on line 28

And Footer should be the last one on the page =)